core/templates/pages/volunteer-page/volunteer-page.component.ts

```diff
@@ -0,0 +1,204 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the volunteer page.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { ViewEncapsulation } from '@angular/core';
+import { NgbCarouselConfig } from '@ng-bootstrap/ng-bootstrap';
+
+import { PageTitleService } from 'services/page-title.service';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+
+
+@Component({
+  selector: 'volunteer-page',
+  templateUrl: './volunteer-page.component.html',
+  styleUrls: [],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
When using view-encapsulation = none, we should wrap the entire component in a div with a class of the selector. Also the selector here should be `oppia-volunteer-page`. All the styles should be prefixed with `.`oppia-volunteer-page`. This is to prevent accidental style leaks into other components.

Reply:
> When using view-encapsulation = none, we should wrap the entire component in a div with a class of the selector.

I have wrapped the component with a class of the selector. Have I missed anything else?

> Also the selector here should be oppia-volunteer-page. All the styles should be prefixed with .oppia-volunteer-page`. This is to prevent accidental style leaks into other components.

I agree with you but none of the other pages' components have the "oppia-" prefix. Adding the prefix only to this page would make things inconsistent. Do you still want me to do it?